go.mod

```diff
@@ -28,11 +28,14 @@ require (
 	sigs.k8s.io/controller-runtime v0.4.0
 )
 
+// Pin the version of client-go to something that's compatible with katrogan's fork of api and apimachinery
```

Comment:
👍

Reply:
Now looks much clearer. To me it has been hard to know whether a comment serves anything in go.